test/test-debuglet.js

```diff
@@ -553,26 +528,21 @@ describe('Debuglet', function() {
       });
       this.timeout(6000);
 
-      var scope = nock(API)
-        .post(REGISTER_PATH)
-        .reply(200, {
-          debuggee: {
-            id: DEBUGGEE_ID
-          }
-        })
-        .get(BPS_PATH + '?success_on_timeout=true')
-        .reply(200, {
-          breakpoints: [bp]
-        })
-        .put(BPS_PATH + '/test', function(body) {
-          return body.breakpoint.status.description.format ===
-            'The snapshot has expired';
-        })
-        .reply(200)
-        .get(BPS_PATH + '?success_on_timeout=true').times(4)
-        .reply(200, {
-          breakpoints: [bp]
-        });
+      var scope =
+          nock(API)
+              .post(REGISTER_PATH)
+              .reply(200, {debuggee: {id: DEBUGGEE_ID}})
+              .get(BPS_PATH + '?success_on_timeout=true')
+              .reply(200, {breakpoints: [bp]})
+              .put(BPS_PATH + '/test',
+                   function(body) {
+                     return body.breakpoint.status.description.format ===
+                            'The snapshot has expired';
+                   })
+              .reply(200)
+              .get(BPS_PATH + '?success_on_timeout=true')
+              .times(4)
+              .reply(200, {breakpoints: [bp]});
```

Comment:
Were these whitespace changes intended? The way they occur in code that was otherwise not changed seems to suggest that they were auto-formatted by the editor.